lib/passport-wsfed-saml2/saml.js

```diff
@@ -93,6 +93,9 @@ SAML.prototype.validateSignature = function (xml, options, callback) {
     if (e.message === 'PEM_read_bio_PUBKEY failed') {
       return callback(new Error('The signing certificate is invalid (' + e.message + ')'));
     }
+    if (e.opensslErrorStack !== undefined) {
+      return callback(new Error('The signing certificate is invalid (' + e.opensslErrorStack.join(', ') + ')'));
```

Comment:
Do you think it'd be valuable to attach the original stack as a property on this object?